model/simulationtests/gshp_desuperheater.rb

```diff
@@ -0,0 +1,59 @@
+require 'openstudio'
+require 'lib/baseline_model'
+
+model = BaselineModel.new
+
+#make a 2 story, 100m X 50m, 10 zone core/perimeter building
+model.add_geometry({"length" => 100,
+                    "width" => 50,
+                    "num_floors" => 2,
+                    "floor_to_floor_height" => 4,
+                    "plenum_height" => 1,
+                    "perimeter_zone_depth" => 3})
+
+#add windows at a 40% window-to-wall ratio
+model.add_windows({"wwr" => 0.4,
+                   "offset" => 1,
+                   "application_type" => "Above Floor"})
+
+#add ASHRAE System type 01, PTAC, Residential
+model.add_hvac({"ashrae_sys_num" => '01'})
+
+#add thermostats
+model.add_thermostats({"heating_setpoint" => 24,
+                       "cooling_setpoint" => 28})
+
+#assign constructions from a local library to the walls/windows/etc. in the model
+model.set_constructions()
+
+#set whole building space type; simplified 90.1-2004 Large Office Whole Building
+model.set_space_type()
+
+#add design days to the model (Chicago)
+model.add_design_days()
+
+#create schedule object
+schedule_constant = OpenStudio::Model::ScheduleConstant.new(model)
+
+#create desuperheater object
+coil_water_heating_desuperheater = OpenStudio::Model::CoilWaterHeatingDesuperheater.new(model, schedule_constant)
+
+#create stratified water heater
+water_heater_stratified = OpenStudio::Model::WaterHeaterStratified.new(model)
```

Comment:
Great, you're testing both GSHP coil _and_ stratified water heater. 👍

Reply:
Can you add a test for multispeed coil too?

Reply:
Yes. And I'm going to rename this test to `test_coil_waterheating_desuperheater_rb`.